db/types/datetime.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.dialects.postgresql.base import TIME as SA_TIME
+
+from db.types import base
+
+TIME_ZONE_DB_TYPE = base.PostgresType.TIME_WITH_TIME_ZONE.value
+WITHOUT_TIME_ZONE_DB_TYPE = base.PostgresType.TIME_WITHOUT_TIME_ZONE.value
+
+
+class TIME_WITHOUT_TIME_ZONE(SA_TIME):
+    def __init__(self, *args, precision=None, **kwargs):
+        # On reflection timezone is passed as a kwarg, so we need to make sure it isn't
+        # included in the TIME init call twice
+        if "timezone" in kwargs:
+            kwargs.pop("timezone")
+        super().__init__(*args, timezone=False, precision=precision, **kwargs)
+
+
+class TIME_WITH_TIME_ZONE(SA_TIME):
```

Comment:
Converting the class to a string still returns `Time` rather than `TIME WITH TIME ZONE`. The `str` representation method should be overridden to return the appropriate value